### PR TITLE
loss function

### PR DESCRIPTION
**Pull request by BarrettLi**
Finish all loss functions.
Please contact "卧床不起" on QQ, if there are any problems.

### CODE REVIEW

**Comment by wangyangting** [comment]
我说怎么会有冲突, 原来是大佬你翻译错位置了.  
你翻译的应该是: http://pytorch.apachecn.org/cn/0.3.0/nn.html#loss-functions 这个地方.  
但是实际上你需要翻译的是: http://pytorch.apachecn.org/cn/0.3.0/nn.html#id41 也就是 torch.nn.functional 下面的内容.  
主要是有两个 function 的地方, 搞错也正常, 这特喵的就非常尴尬了.
